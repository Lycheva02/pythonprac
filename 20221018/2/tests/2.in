:strconc x 'aaa'+x+'qkrq'
:ko 1-1
strconc 'QQ'
ko
strconc 'help'
quit "{} :) {}"