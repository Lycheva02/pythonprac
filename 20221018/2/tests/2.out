aaaQQqkrq
0
aaahelpqkrq
3 :) 6
